hypershift-operator/controllers/nodepool/apiserver-haproxy/haproxy.cfg

```diff
@@ -23,7 +23,7 @@ frontend local_apiserver
 backend remote_apiserver
   mode tcp
   log global
-  option httpchk GET /version
+  option httpchk GET /livez/ping
```

Comment:
We cannot simply switch to `/livez/ping` because it isn't enabled for unauthenticated users by default.